met-web/src/components/engagement/EngagementForm.tsx

```diff
@@ -32,7 +32,7 @@ const EngagementForm = () => {
             fromDate: formatDate(savedEngagement.start_date),
             toDate: formatDate(savedEngagement.end_date),
             description: savedEngagement?.description || '',
-            status_id: savedEngagement?.status_id || '',
```

Comment:
We need to find a way to load the "Draft' status from the engagement_status table, the "status_id" is not what we want to display since its a numeric field (1,2, etc..), using SQLAlchemy there must be a way to configure it so it would join tables as needed